src/components/citizen-list-item/styles.scss

```diff
@@ -25,6 +24,17 @@
     text-overflow: ellipsis;
     overflow: hidden;
     white-space: nowrap;
+
+    @include glass-text-primary-color;
+  }
+
+  &__primary-zid {
+    font-size: 10px;
+    font-style: normal;
```

Comment:
Couple nits: Shouldn't need to set `font-style: normal`. `font-weight: 400` is default so don't need to explicitly set this either.